save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/agent/TestExecutionDto.kt

```diff
@@ -37,5 +42,7 @@ data class TestExecutionDto(
     val expected: Long?,
     val unexpected: Long?,
     val hasDebugInfo: Boolean? = null,
-    val executionId: Long? = null
+    val executionId: Long? = null,
+    val testMetrics: TestMetrics = NoDataAvailable.instance,
```

Comment:
`<Something>Dto` should repeats `<Something>` (entity class) in general.
Can we avoid putting these to Dto, if it's not related to entity too

Reply:
Addressed with d2bb021